tests/smoke-nuget-central-package-management.yaml

```diff
@@ -12,6 +12,8 @@ input:
             repo: dependabot/smoke-tests
             directory: /nuget/central-package-management
             commit: 7dbbef7b02f40ddfa7d32fc3099caf19f370bb9b
+        experiments:
+            nuget_native_analysis: true
```

Comment:
Once the feature flag gets removed from `core`, let's remember to remove it from here as well.

Reply:
Oh, I see you constrained that flagging to a single commit, so that keeps things nice and tidy and easily revertable.